components/Provider/AuthProvider.tsx

```diff
@@ -68,16 +71,19 @@ export const AuthProvider = ({ children }: Props): JSX.Element => {
           return;
         }
 
-        await updateCurrentUserBalance(user.actor);
+        await updateBalances(user.actor);
         setCurrentUser(user);
       };
 
       restore();
     }
   }, []);
 
-  const updateCurrentUserBalance = async (chainAccount: string) => {
-    const balance = await proton.getAccountBalance(chainAccount);
+  const updateBalances = async (chainAccount: string) => {
```

Comment:
should this be 2 functions?

Reply:
Your second "getAccountBalance" call is overwriting what is in the balance variable.

There are two balances now, one that's getAtomicMarketBalance which is the balance that's in the smart contract. This is the amount you have to withdraw, there will only be a balance for this if someone bought your item.

The "getAccountBalance" gets your actual FOOBAR token balance from your wallet and that's the money that's being shown in your Balances on the navbar and what will be spent when you buy things. We need to be able to differentiate these two and only show the banner if there is a balance in the getAtomicMarketBalance version.